themes/default/content/docs/guides/crosswalk/aws/api-gateway.md

```diff
@@ -291,549 +313,452 @@ specifying the `target` property on your route.
 
-```typescript
-import * as awsx from "@pulumi/awsx";
+- The required request parameters in the URI, query string, and headers of an incoming request are
+  included and non-blank.
+- The applicable request payload adheres to the configured JSON schema request model of the method.
 
-const api = new awsx.apigateway.API("myapi", {
-    routes: [{
-        path: "/b",
-        method: "GET",
-        eventHandler: async () => {
-            return {
-                statusCode: 200,
-                body: "<h1>Hello world!</h1>",
-            };
-        },
-        authorizers: [{
-            parameterName: "Authorization",
-            parameterLocation: "header",
-            authType: "oauth2",
-            type: "request",
-            handler: async (event: awsx.apigateway.AuthorizerEvent) => {
-                // Add your own custom authorization logic here.
-                 return awsx.apigateway.authorizerResponse("user", "Allow", event.methodArn);
-            },
-        }],
-    }],
-});
-```
+When enabling validation, you will choose a validation scope, in addition to validation rules.
 
-For additional information about token-based AWS API Gateway Lambda Authorizers, see the
-[AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-use-lambda-authorizer.html).
+### Assigning Validators to APIs and Methods
 
-#### Specifying Your Authorizer's IAM Role
+Validators can be assigned for an entire API or at the individual method level, such as only for `POST` on a given
+route. The validators defined at a method level override any validator set at the global API level.
 
-If your Authorizer requires access to other AWS resources to make its authorization decisions, such as looking
-up entries in a database, you will need to provision the appropriate role for your AWS Lambda Function.
+To enable validation, pass the `requestValidator` property on the API object or individual route. The following
+validator values are available:
 
-Instead of using inline Lambda Functions, you can do this by using `new aws.lambda.CallbackFunction`:
+- `"ALL"`: Validate both the request body and request parameters.
+- `"BODY_ONLY"`: Validate only the request body.
+- `"PARAMS_ONLY"`: Validate only the request parameters.
+
+For example, this enables parameter validation on all routes, and all validation on a specific route:
 
 ```typescript
-import * as aws from "@pulumi/aws";
 import * as awsx from "@pulumi/awsx";
 
-const callbackRole = new aws.iam.Role("callbackRole", {
-    // Specify appropriate AWS permissions here.
-});
-
-const callbackFxn = new aws.lambda.CallbackFunction("callbackFxn", {
-    role: callbackRole,
-    callback: async (event: awsx.apigateway.AuthorizerEvent) => {
-        // --- Add custom authorization logic here. ---
-        return awsx.apigateway.authorizerResponse("user", "Allow", event.methodArn);
+const api = new awsx.apigateway.API("example", {
```

Comment:
This and the next few still needs porting to multi-lang - right?

Reply:
Yup - bashing through the last few sections now